### DISCUSSION TITLE
code navigation doesn't work

### DISCUSSION THREAD

**Discussion by chaeyoungmoon** [community/community#12381] [Other Feature Feedback, Questions, & Ideas]
Hi, 
I'm having a problem with [code navigation](https://docs.github.com/en/repositories/working-with-files/using-files/navigating-code-on-github).

The definition and reference don't show up when I click any functions or methods.
The mouse pointer shape changes into text select shape, not link select shape as I put the curser on these entities.
It worked well before. 
I have tried in private mode and used different browser like edge, but failed in both.
I also rebooted the computer.

It would be appreciated if you could share any ideas about this problem.

**Comment by colinwm** [comment]
Just to clarify, are you referring to code navigation within code search (i.e. cs.github.com)? Or within github.com?

Is it possible that your browser window is too small? We don't render the info box if the browser window is not wide enough to fit it.

**Reply by chaeyoungmoon** [reply]
First of all, thanks for your suggestion.

It’s within just github.com.
Sorry for wrong category. I was confused.

Browser window is wide enough. It worked before and I’m using the same monitor.

Is there anything else I should check?

**Comment by hughetop** [comment]
Is it possible to disable this feature? Function and variable definitions seem to work if they are within the same file, but nearly every python import fails. It is supposed to point to the actual file that is being imported, but it just points to itself. And even when it _does_ work, it seems to link to some random blob instead of the default branch. I'd rather just disable it entirely if it doesn't work properly.
